content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -100,22 +90,16 @@ Cross-cluster communication occurs over Istio gateways of the respective cluster
             --from-file=@samples/certs/cert-chain.pem@
         {{< /text >}}
 
-    * Install all the Istio
-    [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
-    (CRDs) using `kubectl apply`:
-
-    {{< text bash >}}
-    $ helm template install/kubernetes/helm/istio-init --name istio-init --namespace istio-system | kubectl apply -f -
-    {{< /text >}}
-
-    * {{< boilerplate verify-crds >}}
-
-    * Use the Istio installation yaml file generated in a previous step to install Istio:
+    * Install Istio:
 
         {{< text bash >}}
-        $ kubectl apply -f $HOME/istio.yaml
+        $ istioctl manifest apply \
+            -f @install/kubernetes/operator/examples/multicluster/istio-gateway.yaml@
```

Comment:
```suggestion
            -f install/kubernetes/operator/examples/multicluster/istio-gateway.yaml
```
Remove the @ wrapper, for now, because the file doesn't exist yet, so it breaks lint.